src/main/java/games/strategy/engine/data/properties/GameProperties.java

```diff
@@ -41,6 +42,8 @@
   // added.
   private final List<String> ordering = new ArrayList<>();
 
+  private final Map<String, IEditableProperty> playerProperties = new LinkedHashMap<>();
```

Comment:
Store player properties in a separate map so they aren't displayed on settings UI but player UI.